docs/content/users/extend/custom-commands.md

```diff
@@ -218,6 +218,14 @@ If your container command should run from the directory you are running the comm
 
 Example: `## HostWorkingDir: true`
 
+### “ExecRaw” Annotation (Container Commands Only)
+
+Use `ExecRaw: true` to pass arguments for the command you want to run directly to the command in the container as-is. 
+
+For example, when ExecRaw is true, `ddev yarn --help` returns the help for `yarn`, not DDEV's help for the `ddev yarn` command.
```

Comment:
```suggestion
For example, when ExecRaw is true, `ddev yarn --help` returns the help for `yarn`, not DDEV's help for the `ddev yarn` command.

Most commands will work better with this annotation. This is recommended for almost all container commands.
```

Reply:
> Most commands will work better with this annotation. This is recommended for almost all container commands.

Two things:

1. This reads like “you should mostly use this” without qualifying why. Is it possible to give an example of when this is and is not ideal to help someone navigate their own decision?
2. Ideally recommendations can be [active third person](https://ddev.readthedocs.io/en/stable/developers/writing-style-guide/#use-active-third-person); “this is recommended” → “we recommend”.

Reply:
Thanks @mattstein - I updated. But shouldn't that be called "active first person"?